packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -74,15 +84,18 @@ async function orchestratorProcess(
             childArgs.push("--verbose");
         }
 
-        const process = child_process.spawn(
-            "node",
-            childArgs,
-            { stdio: "inherit" },
-        );
-        p.push(new Promise((resolve) => process.on("close", resolve)));
+        console.log(childArgs.join(" "));
+        runnerArgs.push(childArgs);
     }
     try{
-    await Promise.all(p);
+        await Promise.all(runnerArgs.map(async (childArgs)=>{
```

Comment:
What's the reason for this little refactor?

Reply:
to print the runner command lines to make it easier to debug, and i didn't want them interleaved with output, which makes it hard to read